faq.html

```diff
@@ -13,7 +13,7 @@
     const languages = ["{{ languages | join: '", "' }}"]
 
     function switchLanguage(lang) {
-        if ($.inArray(userLang, languages) > -1) {
+        if (languages.includes(lang)) {
```

Comment:
This is easier to understand, but I'm not sure about the browser compatibility. Most modern browsers should support it, though.